### ISSUE TITLE
Translation of templates.

### ISSUE THREAD

**Issue by alexmitrani** [climu/openstudyroom#340]
Discussion of templates to be added and translation issues.

**Comment by alexmitrani** [comment]
Next on the spanish translation list (poeditor):
- Main community page
- League and tournament tabs
-- players
-- games
-- results

thanks

**Comment by alexmitrani** [comment]
We need to enable translation of parts of the website that are in Wagtail.

**Comment by alexmitrani** [comment]
Priority Wagtail pages: 
- FAQ
- Code of conduct
- About OSR

**Comment by climu** [comment]
Yes but that is another issue that require technical work. I would like this one to be related to marking templates to be translate so we keep this organized.

**Comment by climu** [comment]
Community template added. 16 new strings on poeditor. The leagues, tournaments, games, and members tables of the community page are not added yet. Next on the todo.

**Comment by alexmitrani** [comment]
The 16 new strings have been translated

**Comment by climu** [comment]
Translation is on production and 28 new string are to translate on poeditor.
